src/xrGame/ui/UIGameTutorial.cpp

```diff
@@ -97,21 +97,22 @@ bool CUISequenceItem::Stop(bool bForce)
 CUISequencer::CUISequencer() { m_flags.zero(); }
 void CUISequencer::Start(LPCSTR tutor_name)
 {
-    // Skip any tutorial except "game_loaded", since we need to show "st_press_any_key" hint
-    if (load_screen_renderer.IsActive() && xr_strcmp(tutor_name, "game_loaded") != 0)
+    VERIFY(m_sequencer_items.size() == 0);
+
+    CUIXml uiXml;
+    uiXml.Load(CONFIG_PATH, UI_PATH, UI_PATH_DEFAULT, "game_tutorials.xml");
```

Comment:
I believe it should be done in other way to not touch game resources.

Tutorial works for Mixed configuration and it works in original COP, so, it seems that there's something wrong in the our code which has `#ifdef` `MASTER_GOLD`, `DEBUG` or `NDEBUG` macros.